torch/_torch_docs.py

```diff
@@ -3000,13 +2998,23 @@ def merge_dicts(*dicts):
 
     >>> v1 = torch.arange(1., 5.)
     >>> v2 = torch.arange(1., 4.)
-    >>> torch.ger(v1, v2)
+    >>> torch.outer(v1, v2)
     tensor([[  1.,   2.,   3.],
             [  2.,   4.,   6.],
             [  3.,   6.,   9.],
             [  4.,   8.,  12.]])
 """)
 
+add_docstr(torch.ger,
+           r"""
+ger(input, vec2, *, out=None) -> Tensor
+
+Alias of :func:`torch.outer`.
+
+.. warning::
+    This function is deprecated and please use :func:`torch.outer` instead.
```

Comment:
"This function is deprecated. Use :func:\`torch.outer\`, instead."

Reply:
Addressed.